modules/datahandling/datahandler_test.go

```diff
@@ -0,0 +1,181 @@
+package datahandling
```

Comment:
Convention names it after the file it tests. Unless we extract it, it should show which file it's testing